undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
I was reading the docs for `find_module`, and it seems that if you pass in a series of paths, it will search through those for the module, and otherwise will look through `sys.path`.

So, how do you ensure that it looks through the current directory first for the module, and then through `sys.path`? I guess I could temporarily append '.' to `sys.path` and then remove it after?